java/server/src/org/openqa/selenium/grid/component/HealthCheck.java

```diff
@@ -40,5 +48,9 @@ public boolean isAlive() {
     public String getMessage() {
       return message;
     }
+
+    public String registrationSecret() {
```

Comment:
Why is this in the result? It seems like it shouldn't be needed there, as the value will already have been used.